### PR TITLE
Add Highlight.js code highlighting for editor

### PR DESCRIPTION
**Pull request by perrywarner**
See https://highlightjs.org/ for usage information. Currently only highlights HTML/CSS/JS that are put in the editor pane.

TODO: add Markdown highlighting

### CODE REVIEW

File: src/app/services/markdown-service.service.ts [hunk 1]
```diff
@@ -1,6 +1,24 @@
...
+
+const markdownString = '```js\n console.log("hello"); \n```';
+
+// Using async version of marked
+marked(markdownString, function (err, content) {
+  if (err) {throw err; }
+  console.log(content);
+});
+
+// Synchronous highlighting with highlight.js
```
Comment (kstarrmitche) [inline src/app/services/markdown-service.service.ts]: this is useful to know, but I'm not a huge fan of commented out code :'(

File: src/app/services/markdown-service.service.ts [hunk 2]
```diff
@@ -1,6 +1,24 @@
 import { Injectable } from '@angular/core';
 
 import marked from 'marked';
+import highlight from 'highlight.js';
+
+const markdownString = '```js\n console.log("hello"); \n```';
+
+// Using async version of marked
+marked(markdownString, function (err, content) {
+  if (err) {throw err; }
```
Comment (kstarrmitche) [inline src/app/services/markdown-service.service.ts:10]: Good addition! How do I have like no errors/logging in the app...yikes

**Review by kstarrmitche — CHANGES_REQUESTED** [review]
Overall good, but there are a few things I want to see functionality wise:
-  highlight support for more than just css, html, and javascript within the editor pane
-  if not the above, at least additional highlighting for markdown

**Comment by kstarrmitche** [comment]
[Add Syntax Highlighting to markdown editor](https://trello.com/c/qCAy27G6/59-add-syntax-highlighting-to-markdown-editor)

File: src/app/services/markdown-service.service.ts (hunk 1 above)
Comment (perrywarner) [reply src/app/services/markdown-service.service.ts]: fixed this one on the latest commit, not yet pushed. more to come

File: src/app/services/markdown-service.service.ts [hunk 3]
```diff
@@ -11,13 +11,6 @@ marked(markdownString, function (err, content) {
   console.log(content);
 });
 
-// Synchronous highlighting with highlight.js
```
Comment (kstarrmitche) [inline src/app/services/markdown-service.service.ts]: Thank you :)

File: src/styles.css [hunk 4]
```diff
@@ -4,7 +4,7 @@
  * Licensed under MIT (https://github.com/BlackrockDigital/startbootstrap-blog-home/blob/master/LICENSE)
  */
 
- body {
+body {
```
Comment (kstarrmitche) [inline src/styles.css:7]: Good addition! looks better.

**Review by kstarrmitche — COMMENTED** [review]
I'm gonna go ahead and merge in your code! I can tie up any loose ends :)

**Review by kstarrmitche — APPROVED** [review]

File: src/styles.css (hunk 4 above)
Comment (perrywarner) [reply src/styles.css:7]: what a change... lol

Comment (kstarrmitche) [reply src/styles.css:7]: LMAO

Comment (kstarrmitche) [reply src/styles.css:7]: LMAO I thought you had added way more, I'm clearly color blind....

**Pull request merged**